src/components/Static/Downloads/Downloads.js

```diff
@@ -3,27 +3,45 @@ import Template from '../../Template/Template';
 import Card from '../../Card/Card';
 import Button from '../../Button/Button';
 
+
+import EmojiGood from '../../Emoji/good.png';
+
 function Downloads() {
   return (
     <Template>
       <h1 style={{ marginTop: '48px' }}>Downloads</h1>
       <h2>Project Releases</h2>
       <Card title={'Cortex Command Community Project'}>
         <p style={{
-          marginBottom: '18px',
-        }}>This is where you can find the community's plans for Cortex Command's future. There'll be rebalancing, new weapons and units, and much more.</p>
+          marginTop: '8px', marginBottom: '18px',
+        }}>This is where you can find the community's plans for Cortex Command's future. There'll be rebalancing, new weapons, units, and much <i>much</i> more.</p>
+      
         <Button link to='https://github.com/cortex-command-community/Cortex-Command-Community-Project-Data/releases'>Releases</Button>
+      
+        <Button link to='https://github.com/cortex-command-community/Cortex-Command-Community-Project-Data/releases/download/v0.1.0-pre2/CCCP.v0.1.0-pre2.zip'>Pre 2 <img width='14px' height='14px' marginBottom='-12px' src={EmojiGood}/> (latest)</Button>
```

Comment:
`marginBottom` here is giving a console error. It should be defined as part of a style attribute
```style={{ marginBottom= ... }}```